bitnami/postgresql/README.md

```diff
@@ -329,6 +336,35 @@ In addition to these options, you can also set an external ConfigMap with all th
 
 The allowed extensions are `.sh`, `.sql` and `.sql.gz`.
 
+### Securing traffic using TLS
+
+TLS support can be enabled in the chart by specifying the `tls.` parameters while creating a release. The following parameters should be configured to properly enable the TLS support in the chart:
+
+- `tls.enabled`: Enable TLS support. Defaults to `false`
+- `tls.certificatesSecret`: Name of an existing secret that contains the certificates. Alternatively, you can place your `*.crt`, `*.key` and `*.crl` files under the `files/certs` path to create a new secret out of them. No defaults.
+- `tls.certFilename`: Certificate filename. No defaults.
+- `tls.certKeyFilename`: Certificate key filename. No defaults.
+
+For example:
+
+First, create the secret with the cetificates files:
+
+```console
+kubectl create secret generic certificates-tls-secret --from-file=./cert.crt --from-file=./cert.key --from-file=./ca.crt
+```
+
+Then, use the following parameters:
```

Comment:
We could use a list for this (and intent the code) to make it more readable, i.e.:

Refer to the following example:

* First, create the secret with the certificate files:

    ```console
    kubectl create secret ...
    ```

* Then, install the chart with the following parameters:

    ```console
    volumePermissions.enabled=true
    ...

    ```

    > NOTE: PostgreSQL requires certain permissions on sensitive files (such as certificate keys) to start up. Due to an on-going [issue](https://github.com/kubernetes/kubernetes/issues/57923) regarding kubernetes permissions and the use of `securityContext.runAsUser`, you must enable `volumePermissions` to ensure everything works as expected.

___

```
Refer to the following example:

* First, create the secret with the certificate files:

    ```console
    kubectl create secret ...
    ```

* Then, install the chart with the following parameters:

    ```console
    volumePermissions.enabled=true
    ...

    ```

    > NOTE: PostgreSQL requires certain permissions on sensitive files (such as certificate keys) to start up. Due to an on-going [issue](https://github.com/kubernetes/kubernetes/issues/57923) regarding kubernetes permissions and the use of `securityContext.runAsUser`, you must enable `volumePermissions` to ensure everything works as expected.
```

Reply:
Okay, sounds nice to me!